src/blocklet/environments/index.md

```diff
@@ -9,11 +9,49 @@ tags:
   - 'abtnode'
 ---
 
-ABT Node provides a few environment variables to blocklets running on the node.
+Click a Blocklet on the Blocklet Management page to arrive on the configuration page for the respective Blocklet. The page provides details about the installed Blocklet. The page allows you to control Blocklet operation and update its configuration.
 
-![](./images/blocklet-env-1-en.png)
+![](./images/blocklet-details.png)
 
-### System Configuration
+On the details page, there is a row of operation buttons on the top right:
+
+1. `Start`
+2. `Stop`
+3. `Restart`
+4. `Reload`: Reload the configuration
+5. `Remove` (as stated earlier, you will be prompted to choose whether to delete the blocklet data as well)
+
+There are the following three tabs that provide further details about the Blocklet
+
+### Overview Tab
+
+The *Overview* tab provides details like version, status, location, source, etc, for the installed Blocklet binary. It also captures basic usage metrics like uptime, CPU, and memory usage.
+
+![](./images/blocklets-overview.png)
+
+The page also shows different data locations for the different types of files used by the Blocklet.
+- APP Directory: The location to store Blocklet binary.
+- Data Directory: The location to store all data processed by a Blocklet.
+- Log Directory: The location for the logs generated by the Blocklet
+- Cache Directory: The location used for caching all data.
```

Comment:
The location used for caching all data.

=> The location used for caching data by the blocklet.